README.md

```diff
@@ -1,121 +1,70 @@
 # CDF Fabric replicator
 
-Application which utilizes the CDF APIs to replicate data to and from Microsoft Fabric
+Application that utilizes the CDF APIs to replicate data to and from Microsoft Fabric.
 
-# Replicator Services
+## Replicator Services
 The replicator consists of four services:
-- **Time series replicator** - Copies time series data from CDF to Fabric
-- **Data model replicator** - Copies data model nodes and edges from CDF to Fabric
-- **Event replicator** - Copies event data from CDF to Fabric
-- **Fabric data extractor** - Copies time series, events, and files from Fabric to CDF
+- **Time series replicator** - Copies time series data from CDF to Fabric.
+- **Data model replicator** - Copies data model nodes and edges from CDF to Fabric.
+- **Event replicator** - Copies event data from CDF to Fabric.
+- **Fabric data extractor** - Copies time series, events, and files from Fabric to CDF.
 
-All four services will run concurrently during the run of the CDF Fabric Replicator program.  The services use one state store in CDF's raw storage to maintain checkpoints of when the latest data was copied, so the services can be started and stopped and will be able to pick back up where they left off.
+All four services will run concurrently during the execution of the CDF Fabric Replicator program. The services use one state store in CDF's raw storage to maintain checkpoints of when the latest data was copied, so the services can be started and stopped and will be able to pick back up where they left off.
 
-# Setting up Data Point Subscriptions
+## Local Development with CDF Fabric replicator
+Follow these instructions for doing Local Development with CDF Fabric replicator:
```

Comment:
Should we also include steps to create a venv, i think that is needed to run locally

Reply:
The venv setup instructions are explained in the poetry docs but might be worth a mention

Reply:
Added a note to re-enforce the Poetry venv